app/configurations/config.hb.js

```diff
@@ -8,6 +8,7 @@ const API_URL = process.env.API_URL || 'https://api.mobil-in-herrenberg.de';
 const MAP_URL = process.env.MAP_URL || 'https://maps.wikimedia.org/osm-intl/';
 const GEOCODING_BASE_URL = process.env.GEOCODING_BASE_URL || `https://pelias.locationiq.org/v1`;
 const LOCATIONIQ_API_KEY = process.env.LOCATIONIQ_API_KEY;
+const GEOJSON_LAYERS_BASE_URL = process.env.GEOJSON_LAYERS_BASE_URL || '/assets/geojson/hb-layers/';
```

Comment:
Isn't assets/geojson/hb-layers the path to the formerly checked in files? I'd expect the host-relative path defined in digitransit-proxy(?)

Reply:
I modified it but not sure if it's correct. I dont really know what url should fit here (instead of http://opentripplanner-data-con-hb:8080/routing-data/v2/hb/)